pandas/io/formats/style.py

```diff
@@ -1222,6 +1222,78 @@ class MyStyler(cls):
 
         return MyStyler
 
+    def pipe(self, func, *args, **kwargs):
+        """
+        Apply ``func(self, *args, **kwargs)``, and return the result.
```

Comment:
Not exactly true, what happens with the `Styler` instance where this is being called? Don't we modify its result?

Reply:
I'm not sure I understand your second question.

The user-supplied function `func` may or may not modify the styler; it may or may not have side-effects; and it may return the styler, some other result, or None.

Consider the `pandas.Series` case.  A user could write any of the following:

```
>>> s = pandas.Series(...)
>>> s.dropna().shift().pipe(normalize)        # Returns Series
>>> s.dropna().shift().pipe(trimmed_mean)     # Returns scalar
>>> s.dropna().shift().pipe(insert_initial_value)  # Unconventional, not "pand-onic"
>>> s.dropna().shift().pipe(save_to_disk)     # Returns string filename
```